### DISCUSSION TITLE
[Projects Beta] Renaming field should change active filters

### DISCUSSION THREAD

**Discussion by frno** [community/community#6887] [Projects and Issues] [answered]
When renaming a field it currently does not effect active filters, so active filters need to be altered after a field has been renamed. Would be nice if changing a field's name only affected its name and not it's usages.

Reproduction steps:
1. Open project
2. Add a new field to view called "Original" as a Text (Though type does not matter)
3. Add a new item named "Item 1"
4. Populate "Original" field as "One"
5. Create a new view
6. Filter the new view to only show items where Original is set to One. "original:One"
7. Rename the "Original" field
8. New view will now not show any items

**Comment by azenMatt** [comment, marked as answer]
Agreed this is a gap we meed to fix. We are tracking a solution for this but don't have a timeframe for it right now

**Comment by jdshkolnik** [comment]
@azenMatt, it's not just fields. It breaks when anything is renamed, like a milestone.
